numpy/lib/histograms.py

```diff
@@ -204,16 +242,17 @@ def _hist_bin_auto(x):
     --------
     _hist_bin_fd, _hist_bin_sturges
     """
-    fd_bw = _hist_bin_fd(x)
-    sturges_bw = _hist_bin_sturges(x)
+    fd_bw = _hist_bin_fd(x, None)
+    sturges_bw = _hist_bin_sturges(x, None)
```

Comment:
This looks a little weird - better to pass on the `range` argument here, even if it's not used